file.c

```diff
@@ -8,12 +8,27 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <math.h>
 
 #include "buffer.h"
 #include "consts.h"
 #include "highlight.h"
 #include "terminal.h"
 
+int countLines(FILE *fp) {
```

Comment:
```suggestion
int count_lines(FILE *fp) {
```
internal util function